WcaOnRails/lib/cutoff.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+class Cutoff
```

Comment:
Yeah, all that cleverness will be non trivial work you shouldn't do unless needed.